pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -605,6 +607,7 @@ stdenv.mkDerivation (finalAttrs: {
   ] ++ [
     (enableFeature withLadspa "ladspa")
     (enableFeature withLzma "lzma")
+    (enableFeature withMetal "metal")
```

Comment:
Older ffmpeg doesn't know about this flag.

Reply:
enclosed it with optionals